geniza/footnotes/models.py

```diff
@@ -633,6 +633,23 @@ def content_html_str(self):
                 ]
             )
 
+    @cached_property
+    def content_text_canvases(self):
```

Comment:
Could this be generated from the `content_html_str` property? 

Any overlap in functionality/code with how we're currently exporting plain text for the github exports?

Reply:
It's similar, but just different enough in a way that I couldn't figure out how to generalize. It needs to return a list of canvases for solr indexing to prevent the issue with field length exceeding limits. And it's similar to the plaintext export but each canvas needs to have its newlines removed for parity with PGPv3.